include/net/nrf_cloud_coap.h

```diff
@@ -245,6 +275,17 @@ int nrf_cloud_coap_shadow_device_status_update(const struct nrf_cloud_device_sta
  */
 int nrf_cloud_coap_shadow_service_info_update(const struct nrf_cloud_svc_info * const svc_inf);
 
+/**
+ * @brief Send an nRF Cloud Object
```

Comment:
see comment in the c file:
update brief to say what the object requirements are.